platform/pensando/sonic-platform-modules-dpu/dpu/utils/dpu_pensando_util.py

```diff
@@ -0,0 +1,311 @@
+# {C} Copyright 2023 AMD Systems Inc. All rights reserved
+
+import os
+import subprocess
+import sys
+import logging
+import time
+import re
+import json
+from sonic_platform.helper import APIHelper
+
+QSFP_STAT_CTRL_CPLD_ADDR = "0x2"
+apiHelper = APIHelper()
+
+def run_cmd(cmd):
+    status, output = subprocess.getstatusoutput(cmd)
+    if status != 0:
+        try:
+            logger = open("/var/log/dpu_pensando_util.log", "a")
```

Comment:
Why the separate logging file? Please use regular logging infra.

Reply:
done